dspace-uclouvain/src/main/java/org/dspace/uclouvain/submissionMetadataGenerators/generators/model/DegreeMapper.java

```diff
@@ -12,34 +12,28 @@ public class DegreeMapper {
     @JsonProperty("faculty-name")
     private String facultyName;
 
-    public String getRootDegreeCode() {
-        return rootDegreeCode;
-    }
-
+    public String getRootDegreeCode() { return rootDegreeCode; }
```

Comment:
?? I agree with the 2 forms but if we use one we should use it in for all the other method (if not too long on 1 line). :)))

Reply:
When it's possible and line isn't too long(120 chars) it seems a good solution.